v2/loadbalancers.go

```diff
@@ -32,6 +32,7 @@ type LoadBalancerDetails struct {
 type LoadBalancerIP struct {
 	Address         string `json:"ipaddress"`
 	Cost            int    `json:"cost"`
```

Comment:
Will this not fail for IPv6 and Euros being float and not int?

Reply:
Yes, floating point numbers returned will fail here. This was not in scope of the issue. It will be included in the fix for issue #35.